chall02/cvinolo-.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+# **************************************************************************** #
+#                                                                              #
+#                                                         :::      ::::::::    #
+#    cvinolo-.py                                        :+:      :+:    :+:    #
+#                                                     +:+ +:+         +:+      #
+#    By: cvinolo- <cvinolo-@student.42.fr>          +#+  +:+       +#+         #
+#                                                 +#+#+#+#+#+   +#+            #
+#    Created: 2020/04/06 18:54:53 by cvinolo-          #+#    #+#              #
+#    Updated: 2020/04/06 18:54:53 by cvinolo-         ###   ########.fr        #
+#                                                                              #
+# **************************************************************************** #
+
+#
+# Constelación Serpens (Chall02) / String to Morse
+#
+#Implemented shebang to indicate (in unix system)
+#where the environment variable "python3" is.
+#
+#Solved with a dictionary to list morse alphabet with a
+#key, value pair based on modern english alphabet
+#(including space character for a more friendly implementation).
+#
+#Defined functions to obtain the command usage and the
+#morse message, in order to separate the logic from the visual.
+#
+#Implemented try/except to handling all possibles errors and
+#exceptions we don't want in our program.
+#
+
+import sys
+
+morse_alphabet = {
+"a": ".-",
+"b": "-...",
+"c": "-.-.",
+"d": "-..",
+"e": ".",
+"f": "..-.",
+"g": "--.",
+"h": "....",
+"i": "..",
+"j": ".---",
+"k": "-.-",
+"l": ".-..",
+"m": "--",
+"n": "-.",
+"o": "---",
+"p": ".--.",
+"q": "--.-",
+"r": ".-.",
+"s": "...",
+"t": "-",
+"u": "..-",
+"v": "...-",
+"w": ".--",
+"x": "-..-",
+"y": "-.--",
+"z": "--..",
+" ": " "
+}
+
+def get_usage():
+	return "usage: ./cvinolo-.py <a-zA-Z string>"
+
+def get_morse_message(message):
+	morse_message = ""
+	message = message.lower()
+	try:
```

Comment:
It is better to use conditional check as you did line `77`
```python
if "blah" in B:
    A["blah"] = B["blah"]
```
 than KeyError exception for dictionary key 
```python
try:
    A["blah"] = B["blah"]
except KeyError:
    pass
```

You didn't specified the exceptions and it will catch all of them